### PR TITLE
Fix OCPQE-9578

### PR DESCRIPTION
**Pull request by redHatGong**
Fix issue in [OCPQE-9578](https://issues.redhat.com/browse/OCPQE-9578)
Error description
1. When restoring SCC restricted-v2 in 4.11, need to delete uid & resourceVersion
2. `oc create -f` has race condition with the cluster's automatic recreation. so using `oc replace -f`

Local cucumber test log pastebin in 4.11: 1056926
Local cucumber test log pastebin in 4.10: 1056939

self review is done, please help CR, thanks a lot. 
@xingxingxia @y4sht

### CODE REVIEW

File: features/step_definitions/scc.rb [hunk 1]
```diff
@@ -36,12 +36,15 @@
...
   end
-
+  orig_policy = @result[:parsed]
+  orig_policy['metadata'].delete('resourceVersion')
+  orig_policy['metadata'].delete('uid')
+  orig_policy = orig_policy.to_json
   _admin = admin
   teardown_add {
-    step %Q/admin ensures "#{policy}" scc is deleted/
+    # `oc create -f` has race condition with the cluster's automatic recreation. so use `oc replace -f` here
```
[the remark this answers is not in the document]
Comment (xingxingxia) [reply features/step_definitions/scc.rb:45]: Actually, you should think to whether need to do changes **around**, when new change happens. Given you made new change of removing line 42, line 45 comment related to line 42 is not needed at all.

**Comment by redHatGong** [comment]
Modifications have been made based on your suggestion, please help CR, thanks a lot. @xingxingxia @y4sht

**Comment by xingxingxia** [comment]
/lgtm
/assign @liangxia
Thanks

**Pull request merged**